features/Journalist_can_create_article.feature

```diff
@@ -21,6 +21,7 @@ Feature: Journalist can create article
         Given I fill in 'Title' field with 'My article'
         And I fill in 'Description' field with 'This is a description'
         And I fill in 'Content' field with 'This is my article about the sunny weather'
+        And I attach a file
```

Comment:
Add this step to the `Journalist_can_update_article` feature test as well

Reply:
It's been updated